Resources/doc/cookbook/recipe_data_mapper.rst

```diff
@@ -0,0 +1,119 @@
+Using DataMapper to work with domain entities without per field setters
+=======================================================================
+
+This is an example of using DataMapper with entities that lack setters/getters for each field.
```

Comment:
`s/lack/avoid` => this will suggest not having setters and getters for each field is a good thing, which I think it is.